Cargo.toml

```diff
@@ -33,6 +34,14 @@ default-features = false
 features = ["suggestions", "color", "wrap_help"]
 version = "2"
 
+[dependencies.diesel]
```

Comment:
Does `optional = true` for the server dependencies means these dependencies will only compile for `--features server`? Should the diesel dependencies also use this?